deis-dev/manifests/deis-builder-rc.yaml

```diff
@@ -37,7 +37,13 @@ spec:
             - name: minio-user
               mountPath: /var/run/secrets/object/store
               readOnly: true
+            - name: builder-key-auth
+              mountPath: /var/run/secrets/api/auth
```

Comment:
It can be, `builder-api` feels weird since it is not the API for builder :)

It expands to `/var/run/secrets/api/auth/builder-key` FYI